stream_alert/alert_processor/outputs/credentials/provider.py

```diff
@@ -0,0 +1,654 @@
+"""
+Copyright 2017-present, Airbnb Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+import json
+import os
+import shutil
+import tempfile
+from abc import abstractmethod
+
+import boto3
+from botocore.exceptions import ClientError
+
+from stream_alert.shared.helpers.boto import default_config
+from stream_alert.shared.logger import get_logger
+from stream_alert_cli.outputs.helpers import kms_encrypt, send_creds_to_s3
```

Comment:
this import cannot be done here, as this will break when packaged and sent to lamdba. essentially, nothing from the `stream_alert_cli` package should ever be imported into the `stream_alert` module..

if this is something we need to share, you can move to somewhere within the `stream_alert/shared` package, as this _is_ bundled for deployment to lambda. then you can use import `stream_alert.shared` from the `stream_alert_cli` package if need-be

Reply:
ah.... I was wondering why there was duplicated code in the `helpers` module and the old Dispatcher code. TIL